cms/release_calendar/utils.py

```diff
@@ -0,0 +1,54 @@
+from datetime import date
+
+FULL_ENGLISH_MONTHS = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+]
+
+FULL_WELSH_MONTHS = [
+    "Ionawr",
+    "Chwefror",
+    "Mawrth",
+    "Ebrill",
+    "Mai",
+    "Mehefin",
+    "Gorffennaf",
+    "Awst",
+    "Medi",
+    "Hydref",
+    "Tachwedd",
+    "Rhagfyr",
+]
+
+DATE_MONTHS = {
+    "en": FULL_ENGLISH_MONTHS,
+    "cy": FULL_WELSH_MONTHS,
+}
+
+
+def parse_month_year(text: str, locale_code: str) -> date | None:
```

Comment:
Question: It looks like Python's [`calendar`](https://docs.python.org/3/library/calendar.html) module has some utils and constants which might be helpful here.

It claims to support locales too, which could be especially helpful.